### ISSUE TITLE
Request for app to be able to export as GPX

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/android#107]
https://twitter.com/LogicalSlow/status/602928991322787840

This is most easily achieved by the app simply loading the GPX file at the relevant URL. We'll make this into a formal API location and let you know so it can be added as a feature in the app.

**Comment by oliverlockwood** [comment]
I note that @andreig91 made a branch on his fork to add some of this functionality.  I'm not sure why it was never raised as a PR 😄 
I've copied his branch across (see e.g. this commit: https://github.com/cyclestreets/android/commit/9a4093738d04557d5afb0a348c0e469334acc033).

@mvl22 are you happy for work to proceed on this, based on requesting GPX files from the servers from URLs like https://cyclestreets.net/journey/52759941/cyclestreets52759941balanced.gpx ?

**Comment by mvl22** [comment]
> @mvl22 are you happy for work to proceed on this, based on requesting GPX files from the servers from URLs like https://cyclestreets.net/journey/52759941/cyclestreets52759941balanced.gpx ?

Yes. I'm trying to get round to having GPX as an output type from the API, rather than this rather workaroundy kind of URL, but it's fine to use for now. We can just change it in future.
